chats/apps/sectorqueue/models.py

```diff
@@ -0,0 +1,100 @@
+from django.db import models
+from chats.core.models import BaseModel
+from django.utils.translation import gettext_lazy as _
+from chats.apps.sectors.models import Sector, SectorAuthorization
+
+
+class SectorQueue(BaseModel):
+    sector = models.ForeignKey(
+        Sector,
+        verbose_name=_("queues"),
+        related_name="sector_authorizations",
```

Comment:
Here it should be `queues`.

Reply:
I change the model name do Queue and remade the migrations, leaving only the initial with the correct name.